DPNUnet.imjoy.html

```diff
@@ -94,14 +93,11 @@
 from scipy.misc import imread
 import base64
 from shutil import copyfile
-
+from PIL import Image
+from io import BytesIO
 import argparse
-
-import os
```

Comment:
We should keep one `import os`

Reply:
the `import os ` already exist in the line 51 .
so i delete it .